packages/web-components/src/components/va-text-input/va-text-input.css

```diff
@@ -58,6 +58,6 @@ input:not([disabled]):focus {
   border: 2px solid var(--color-secondary-dark);
 }
 
-:host([status='success']) input {
+:host([success='true']) input {
```

Comment:
Could we use the [`:valid` pseudo-selector](https://developer.mozilla.org/en-US/docs/Web/CSS/:valid) to apply the styling?

Reply:
We could update it to `:valid`, but if the input isn't required it will display a green border. This would be a design change for any `va-text-input` that is optional. If we do want this, can we implement it or do we have to run it by DSC?

https://css-tricks.com/almanac/selectors/v/valid/#:~:text=The%20%3Avalid%20selector%20allows%20you,an%20evaluation%20of%20user%20input.

Reply:
It seems like we may want to add an additional indicator of the status besides color - maybe an icon?

https://developer.mozilla.org/en-US/docs/Web/CSS/:valid#accessibility_concerns

Reply:
Interesting, @andresriveratoro. I assume you are referring to the success state? The error state has the error message which is an indicator.

Reply:
Exactly. Our error states (in this component and others) seem fine to me because they include text/icon in addition to the border color change, but for the success state we have just the border.

Reply:
We don't have to deal with `:valid` here since it sounds like there's more design discussion to be had.